examples/misc_camera_traveling.html

```diff
@@ -0,0 +1,187 @@
+            // This layer is defined in a json file but it could be defined as a plain js
+            // object. See Layer for more info.
+            itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(function _(config) {
+                config.source = new itowns.WMTSSource(config.source);
+                const orthoLayer = new itowns.ColorLayer('Ortho', config);
+                view.addLayer(orthoLayer);
+            });
+
+            // Add two elevation layers.
+            // These will deform iTowns globe geometry to represent terrain elevation.
+            function addElevationLayerFromConfig(config) {
+                config.source = new itowns.WMTSSource(config.source);
+                let layer = new itowns.ElevationLayer(config.id, config);
+                view.addLayer(layer);
+            }
+            itowns.Fetcher.json('./layers/JSONLayers/WORLD_DTM.json').then(addElevationLayerFromConfig);
+            itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addElevationLayerFromConfig);
+
+
+            // THREE.Group which stores picked points markers.
+            let pickedPointsGroup = new itowns.THREE.Group();
+            pickedPointsGroup.visible = false;
+            view.scene.add(pickedPointsGroup);
+
+            // Add a pointer at a given position.
+            function addMeshToScene(position) {
+                // Copy the view camera3D to modify its far plane distance.
+                // The will render a CameraHelper with shorter axes.
+                const copyCamera = view.camera.camera3D.clone();
+                copyCamera.far = 50;
+                // Creation of the new mesh :
+                const mesh = new itowns.THREE.CameraHelper(copyCamera);
+                mesh.lookAt(position);
+                mesh.updateMatrixWorld();
+                // Add mesh to scene :
+                pickedPointsGroup.add(mesh);
+                view.notifyChange();
+            }
+
+            // Remove all meshes from scene.
+            function clearMeshes() {
+                pickedPointsGroup.remove(...pickedPointsGroup.children);
+                view.notifyChange();
+            }
+
+            // Array which stores picked camera positions, headings, tilts...
+            let cameraTravel = [];
+            // Camera speed in meters per second
+            let cameraSpeed = 500 / 3.6;
+            // Boolean which states if the camera is performing a travel animation or not
+            let travelOn = 0;
+
+            function saveCurrentCameraTransformOptions() {
+                // Retrieve camera current transform options
+                const cameraTransformOptions = itowns.CameraUtils.getTransformCameraLookingAtTarget(
+                    view,
+                    view.camera.camera3D,
+                );
+                // Save camera current transform options
+                cameraTransformOptions.position = view.camera.camera3D.position.clone();
```

Comment:
not necessary, clone the camera instead

Reply:
I'm not sure I understand what you mean. I believe it is mandatory to store full `CameraTransformOptions` because, when animating camera with `CameraUtils.sequenceAnimationToLookAtTarget`, the `heading` and `tilt` properties from the `CameraTransformOptions` are used to orient the animated camera.